beets/plugins.py

```diff
@@ -694,8 +733,9 @@ def candidates(self, items, artist, album, va_likely):
         query_filters = {'album': album}
         if not va_likely:
             query_filters['artist'] = artist
-        albums = self._search_api(query_type='album', filters=query_filters)
-        return [self.album_for_id(album_id=a['id']) for a in albums]
+        results = self._search_api(query_type='album', filters=query_filters)
+        albums = [self.album_for_id(album_id=r['id']) for r in results]
+        return [a for a in albums if a is not None]
```

Comment:
This bit logic along with [this](https://github.com/beetbox/beets/pull/3389/files#diff-f4ce7fc73415332dc2333633555c3f06R87) early return were added to exclude "empty" (0-track) albums I encountered such as https://www.deezer.com/us/album/6979394.